jellyfin-core/src/jvmCommonMain/kotlin/org/jellyfin/sdk/compatibility/JavaContinuation.kt

```diff
@@ -0,0 +1,16 @@
+package org.jellyfin.sdk.compatibility
+
+import kotlin.coroutines.Continuation
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+public abstract class JavaContinuation<T> : Continuation<T> {
+	override val context: CoroutineContext = EmptyCoroutineContext
+	override fun resumeWith(result: Result<T>): Unit = result.fold(
+		onSuccess = ::onSuccess,
+		onFailure = ::onError,
+	)
+
+	public abstract fun onSuccess(response: T)
```

Comment:
```suggestion
	public abstract fun onSuccess(value: T)
```

or "result" or something different.